demo-shell-ng2/src/tsconfig.dev.json

```diff
@@ -5,7 +5,7 @@
     "module": "es2015",
     "rootDir": "..",
     "baseUrl": ".",
-    "skipLibCheck": true,
+    "skipLibCheck": false,
```

Comment:
we can remove this property. The default value is "false" and usually it is not present in the tsconfig file unless you want to set it to "true"